symphony-bdk-http/symphony-bdk-http-api/src/main/java/com/symphony/bdk/http/api/tracing/DistributedTracingContext.java

```diff
@@ -0,0 +1,59 @@
+package com.symphony.bdk.http.api.tracing;
+
+import org.apiguardian.api.API;
+import org.slf4j.MDC;
+
+import java.util.Random;
+
+/**
+ * Helper class that manipulates underlying logger {@link MDC} for distributed tracing purpose. The Symphony platform
+ * internally relies on header <code>X-Trace-Id</code> value that is printed at server side logs. This is especially
+ * useful when debugging issues across multiple distributed systems.
+ */
+@API(status = API.Status.STABLE)
+public final class DistributedTracingContext {
+
+  /** {@link Random} is thread-safe */
+  private static final Random RANDOM = new Random();
```

Comment:
security scanners like checkmarx will complain that it is not a SecureRandom

Reply:
Fixed in https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/pull/361/commits/80d3ba38edf9684574851a186380181539c319f9